explorer/src/components/Staking/OperatorsList.tsx

```diff
@@ -230,7 +274,40 @@ export const OperatorsList: FC = () => {
           </div>
         ),
       },
-    ]
+    )
+    if (useRpcData && deposits.find((d) => d.account === subspaceAccount))
+      cols.push({
+        accessorKey: 'myStake',
+        header: 'My Stake',
+        enableSorting: !useRpcData,
+        cell: ({
+          row,
+        }: Cell<
+          OperatorsConnectionQuery['operatorsConnection']['edges'][0]['node'] | Operators
+        >) => {
+          const deposit = deposits.find(
+            (d) => d.account === subspaceAccount && d.operatorId.toString() === row.original.id,
+          )
+          const op = rpcOperators.find((o) => o.id === row.original.id)
+          const sharesValue =
+            op && BigInt(op.currentTotalShares) > BigInt(0)
+              ? (BigInt(op.currentTotalStake) * BigInt(1000)) / BigInt(op.currentTotalShares)
+              : BigInt(0)
+          return (
+            <div>
+              {deposit && deposit.shares !== '0' && (
+                <>
+                  {`Staked: ${bigNumberToNumber(((BigInt(deposit.shares) * BigInt(sharesValue)) / BigInt(1000)).toString())} ${selectedChain.token.symbol}`}
```

Comment:
`sharesValue` should already be a `bigint`. 
```suggestion
                  {`Staked: ${bigNumberToNumber(((BigInt(deposit.shares) * sharesValue) / BigInt(1000)).toString())} ${selectedChain.token.symbol}`}
```